.github/workflows/report-on-vulnerabilities.yaml

```diff
@@ -27,13 +27,14 @@ jobs:
       id: parse-results
       continue-on-error: true
       run: |
-        VULNS=$(cat scan.json | jq '.Results[] | has("Vulnerabilities")')
-        if echo $VULNS | grep -q 'true'; then
-          echo "Vulnerabilities found, creating issue"
-          echo "results=$(cat scan.json)" >> $GITHUB_OUTPUT
-        else
+        VULNS=$(cat scan.json | jq '.Results[] | select(.Target=="ko-app/kyverno") | length')
+        if [[ $VULNS -eq 0 ]]
+        then
           echo "No vulnerabilities found, halting"
           echo "results=nothing" >> $GITHUB_OUTPUT
+        else
+          echo "Vulnerabilities found, creating issue"
+          echo "results=$(cat scan.json)" >> $GITHUB_OUTPUT
```

Comment:
is this going to work ? the `scan.json` contains double quotes.

Reply:
It worked before. Why would it not work, you think?

Reply:
Dunno why it worked before but don’t we need to escape the content of the file ?

Reply:
Removing those lines.

Reply:
Shit. I can't remove those lines.

Reply:
i think you can set them to a fixed value. next job will download the file anyway...

Reply:
@chipzoller you need to output something because of the conditions `if: contains(needs.scan.outputs.results, 'SchemaVersion')`

Reply:
Try that. I compacted the output with `jq`.

Reply:
It's useless. All you need is an output to create (or not) issues.